src/Webapi/Dom/Webapi__Dom__HtmlElement.res

```diff
@@ -5,7 +5,7 @@ module Impl = (
 ) => {
   type t_htmlElement = T.t
 
-  let ofElement = Webapi__Dom__Element.asHtmlElement
+  let ofElement: Dom.element => option<t_htmlElement> = Obj.magic(Webapi__Dom__Element.asHtmlElement)
```

Comment:
What about checking `tagName`? seems well supported: https://developer.mozilla.org/en-US/docs/Web/API/Element/tagName

Reply:
I also don't want to have such a central method switch on known HTML element tag names - but we could perhaps build a new `classify` function to do that, since tree shaking will omit it if it isn't used.